src/styles/index.scss

```diff
@@ -1,3 +1,73 @@
+$cell-width: 100px;
+$cell-height: 100px;
+$days: (
+  'mon': 0,
+  'tue': 1,
+  'wed': 2,
+  'thu': 3,
+  'fri': 4,
+  'sat': 5,
+  'sun': 6,
+);
+
+html {
+  font-family: Arial, sans-serif;
+  font-size: 30px;
+}
+
 body {
   margin: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+.calendar {
+  flex-wrap: wrap;
+  gap: 1px;
+  width: calc(7 * $cell-width + 6 * 1px);
+  padding: 0 10px;
+  display: flex;
+
+  &__day {
+    background-color: #eee;
+    width: $cell-width;
+    height: $cell-height;
+    border: 1px solid black;
+    box-sizing: border-box;
```

Comment:
It's good practice to list 'box-sizing' before width and height properties to indicate that the box model is being altered before setting the dimensions as per the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules).